Subject: Update org.jsoup:jsoup to 1.23.1 for CVE remediation
Origin: DHI-internal CVE remediation
Reference: https://repo.maven.apache.org/maven2/org/jsoup/jsoup/1.23.1/
Description: Gradle 9.7.1 pins jsoup 1.22.2 in shared dependency-management
  versions. Bump to 1.23.1, the first release that fixes CVE-2026-71497.

diff --git a/gradle/dependency-management/build.versions.toml b/gradle/dependency-management/build.versions.toml
index 26822af4..6ae569bc 100644
--- a/gradle/dependency-management/build.versions.toml
+++ b/gradle/dependency-management/build.versions.toml
@@ -50,7 +50,7 @@ javaPoet = { module = "com.squareup:javapoet", version.ref = "javaPoet" }
 jaxb = { module = "com.sun.xml.bind:jaxb-impl", version = "4.0.5" }
 jhighlight = { module = "com.uwyn:jhighlight", version = "1.0" }
 jmhPlugin = { module = "me.champeau.jmh:jmh-gradle-plugin", version = "0.7.3" }
-jsoup = { module = "org.jsoup:jsoup", version = "1.22.2" }
+jsoup = { module = "org.jsoup:jsoup", version = "1.23.1" }
 jspecify = { module = "org.jspecify:jspecify", version = "1.0.0" }
 junit = { group = "junit", name = "junit", version = "4.13.2!!" } # keep JUnit 4 transitive we have sometimes up to date
 junitBom = { group = "org.junit", name = "junit-bom", version.ref = "junit" }
diff --git a/gradle/dependency-management/distribution.versions.toml b/gradle/dependency-management/distribution.versions.toml
index 3a4d1885..82229ce8 100644
--- a/gradle/dependency-management/distribution.versions.toml
+++ b/gradle/dependency-management/distribution.versions.toml
@@ -32,7 +32,7 @@ jnr = "0.10.4!!"
 joda = "2.14.0!!"
 jquery = "3.7.1!!" # used in resoures
 jsch = "0.2.16!!"
-jsoup = "1.22.2!!"
+jsoup = "1.23.1!!"
 jspecify = "1.0.0!!"
 jsr305 = "3.0.2!!"
 kotlin = "2.4.0!!"
